assets/js/modules/search-console/components/dashboard-details/DashboardDetailsWidgetKeywordsTable.js

```diff
@@ -32,18 +32,23 @@ import Layout from '../../../../components/layout/layout';
 import { STORE_NAME } from '../../datastore/constants';
 import { STORE_NAME as CORE_SITE } from '../../../../googlesitekit/datastore/site/constants';
 import { getCurrentDateRangeDayCount } from '../../../../util/date-range';
+import { unTrailingSlashIt } from '../../../../util';
 
 const { useSelect } = Data;
 
 const DashboardDetailsWidgetKeywordsTable = () => {
 	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
 	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const isDomainProperty = useSelect( ( select ) => select( STORE_NAME ).isDomainProperty() );
+	const referenceSiteURL = useSelect( ( select ) => unTrailingSlashIt( select( CORE_SITE ).getReferenceSiteURL() ) );
```

Comment:
Not a big deal, but if we're going to wrap the result of selectors, lets make it not a one-liner to improve readability. This is pretty dense code. How about:
```suggestion
	const referenceSiteURL = useSelect( ( select ) => {
		return unTrailingSlashIt( select( CORE_SITE ).getReferenceSiteURL() );
	} );
```